forest/src/logger/mod.rs

```diff
@@ -1,9 +1,12 @@
 // Copyright 2019-2022 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
 
+use crate::cli::config::LogConfig;
+use crate::cli_error_and_die;
 use log::LevelFilter;
+use std::str::FromStr;
 
-pub(crate) fn setup_logger() {
+pub(crate) fn setup_logger(option_log_config: Option<LogConfig>) {
```

Comment:
A reference to `LogConfig` is enough, right? That way we don't need to clone it from the configuration.

```suggestion
pub(crate) fn setup_logger(option_log_config: Option<&LogConfig>) {
```

Reply:
In this case we need to introduce a lifetime, so cloning is better, I guess

Reply:
I don't think you need to introduce a lifetime.

Reply:
Fixed

Reply:
Have you pushed your code? I don't see it being fixed.